### PR TITLE
Fixes #12674 - assign org to host at registration time if not in org

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
There is a current pipeline failure due to this

File: app/models/katello/host/subscription_facet.rb [hunk 1]
```diff
@@ -78,9 +79,9 @@ def update_facts(rhsm_facts)
       def self.find_host(name, organization)
         hosts = ::Host.where(:name => name)
         return nil if hosts.empty? #no host exists
-        if hosts.where(:organization_id => organization.id).empty? #not in the correct org
-          #TODO http://projects.theforeman.org/issues/11532
-          fail "Can't handle registering to host in a different org, need to handle this case."
+        if hosts.where("organization_id = #{organization.id} OR organization_id is NULL").empty? #not in the correct org
+          #TODO: http://projects.theforeman.org/issues/11532
+          fail _("Host is currently registered to a different org, please migrate host to %s.", organization.name)
```
Comment (daviddavis) [inline app/models/katello/host/subscription_facet.rb]: This string interpolation looks off.

**Comment by daviddavis** [comment]
ACK pending small comment.

File: app/models/katello/host/subscription_facet.rb (hunk 1 above)
Comment (jlsherrill) [inline app/models/katello/host/subscription_facet.rb]: I was just inventing a new interpolation method.  Its not my fault ruby hasn't implemented it yet

**Pull request merged**